### PR TITLE
Integration with Travis and coveralls

### PR DESCRIPTION
**Pull request by ywangii**
Here we are:
screen shot 2018-05-16 at 4 43 52 pm

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
```
Comment (jcrugzz) [inline .travis.yml:2]: remove 9, add 10

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  - "9"
+  - "8"
```
Comment (gergelyke) [inline .travis.yml]: can you add 10?

**Review by jcrugzz — CHANGES_REQUESTED** [review]

**Review by gergelyke — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (ywangii) [reply .travis.yml:2]: yes, TIL, copy/paste is bad. XD

File: .travis.yml (hunk 2 above)
Comment (ywangii) [reply .travis.yml]: updated! thanks for quick review sir!

**Review by jcrugzz — APPROVED** [review]

**Pull request merged**